numpyro/compat/ops.py

```diff
@@ -1,3 +1,34 @@
+import numpy as onp
+
+import jax.numpy as np
 from jax.numpy import *  # noqa: F401, F403
 
 tensor = array  # noqa: F405
+
+randn = onp.random.randn
+
+
+# Provide wrappers to initialize ones/zeros using the pytorch convention
```

Comment:
I think it's fine to restrict to numpy-style usage in pyro-api, but if you want to maintain these that's fine too. Let me know either way, and we can switch the pyroapi.tests to use numpy-style invocations.

Reply:
I don't mind maintaining these wrappers, my hope is that this will be relatively hassle free, lets see. :smile: There is a bit of an inconsistency in the numpy API too, because `torch.random.randn()` doesn't accept a tuple size arg, so this provides a more consistent interface.